docs/src/pages/[platform]/components/tabs/demo.tsx

```diff
@@ -9,7 +9,6 @@ import { demoState } from '@/utils/demoState';
 const propsToCode = (props) => {
   return (
     `<Tabs` +
-    `\n  currentIndex="${props.currentIndex}"` +
```

Comment:
I do think the dropdown is little confusing, but it's nice to demo the `currentIndex` prop. Is it a better way to demo it?

Reply:
No we do not want to make it controlled. If you copy the code, it will not work out of the box.